eng/Generate.ps1

```diff
@@ -141,11 +140,11 @@ if ($updateLaunchSettings)
         $definition = $swaggerDefinitions[$key];
         $outputPath = (Join-Path $definition.output $key).Replace($repoRoot, '$(SolutionDir)')
         $codeModel = Join-Path $outputPath 'CodeModel.yaml'
-        $namespace = $definition.title.Replace('-', '_')
+        $namespace = $definition.projectName.Replace('-', '_')
 
         $settings.profiles[$key] = [ordered]@{
             'commandName'='Project';
-            'commandLineArgs'="--standalone --input-codemodel=$codeModel --plugin=csharpgen --output-folder=$outputPath --namespace=$namespace --shared-source-folder=$sharedSourceNormalized --save-code-model=true"
+            'commandLineArgs'="--standalone $outputPath"
```

Comment:
Looking at this change here finally made me realize why this PR seems odd. So, the command line argument for AutoRest are simply AutoRest configuration values. Meaning, `standalone`, `input-codemodel`, `plugin`, `output-folder`, `namespace`, `shared-source-folder`, and `save-code-model` are all values that can be put into a `readme.md`.

This PR invents another configuration file, `Configuration.json`, that serves the same purpose of the configuration in an AutoRest `readme.md`. All the values in this new `Configuration.json` are standard AutoRest values other than `SharedSourceFolder`. But, you can simply make a value `shared-source-folder` in the `readme.md` and read it in our plugin.

So, while I think it is a good idea for each sample to have it's own configuration, we should just use `readme.md` for that purpose. That's the point of AutoRest supporting configuration in this way.